core/pom.xml

```diff
@@ -279,7 +280,7 @@
                                 <include>rest-api-spec/**/*</include>
                                 <include>org/elasticsearch/test/**/*</include>
                                 <include>org/elasticsearch/bootstrap/BootstrapForTesting.class</include>
-                                <include>org/elasticsearch/bootstrap/XTestSecurityManager*.class</include>
+                                <include>org/elasticsearch/bootstrap/MockPluginPolicy.class</include>
```

Comment:
its not doubled up: currently these things are listed twice, once for the stuff that creates the sources jar, the other for the binary jar. We should really try to clean that up!